### PR TITLE
Docs improvements in README.md

### PR DESCRIPTION
**Pull request by spier**
Documentation fixes in the main README.md.

## Proposed Changes

- Some rewording in the first sentence, highlighting that the output of the GHA is a GitHub issue.
- minor bug fix (double space)
- minor cosmetics fixes in the markdown (not visible for the general readers, just for people reading the markdown)
- I am also leaving one question as a comment in this PR, about an area where I am not sure how the `*` footnotes are supposed to be formatted

### Author/Contributor

- [x] If documentation is needed for this change, has that been included in this pull request
- [x] run `make lint` and fix any issues that you have introduced
- [x] run `make test` and ensure you have test coverage for the lines you are introducing
- [x] Label as either `bug`, `documentation`, `enhancement`, `infrastructure`, or `breaking`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +21,16 @@
...
 | Metric | Description |
 |--------|-------------|
 | Time to first response | The time between when an issue/pull request/discussion is created and when the first comment or review is made.* |
 | Time to close | The time between when an issue/pull request/discussion is created and when it is closed.* |
 | Time to answer | (Discussions only) The time between when a discussion is created and when it is answered. |
-| Time in label | The time between when a label has a specific label applied to an issue/pull request/discussion and when it is removed. This requires the LABELS_TO_MEASURE env variable to be set. |
+| Time in label | The time between when a label has a specific label applied to an issue/pull request/discussion and when it is removed. This requires the `LABELS_TO_MEASURE` env variable to be set. |
 
 *For pull requests, these metrics exclude the time the PR was in draft mode.
 *For Issue and pull requests, issue/pull request author's own comments and comments by bots are excluded.
```
Comment (spier) [inline README.md:36]: I assume that these are the footnotes, explaining more details about the `*` in row 1 and row 2 of the table above?

I was not sure:
- if these are two separate footnotes, or if both footnotes apply to both rows?
- if the footnotes are supposed to be rendered in a single row (as they are now)? it looks a bit strange this way.

Screenshot:

Screenshot 2023-10-10 at 23 51 24

**Comment by spier** [comment]
I cannot add a label, as requested in the instructions.

**Comment by spier** [comment]
In order to `make lint` as the instructions request, I had to `pip3 install flake8 pylint`.

Not sure if those two dependencies are missing from `requirements.txt`?
(maybe not, if they are only required when developing/testing?)

**Comment by zkoppert** [comment]
> I cannot add a label, as requested in the instructions.

Thanks for catching this!
Fix this here: https://github.com/github/issue-metrics/commit/3dd3d58dbe49702eef6e87ae031e832044742fcd

File: README.md (hunk 1 above)
Comment (zkoppert) [reply README.md:36]: Yeah, these are supposed to be rendered in separate rows 😓

**Comment by zkoppert** [comment]
> In order to `make lint` as the instructions request, I had to `pip3 install flake8 pylint`.
> 
> Not sure if those two dependencies are missing from `requirements.txt`? (maybe not, if they are only required when developing/testing?)

I've got a [pull request](https://github.com/github/issue-metrics/pull/138) open to fix this. Another great find. Thanks for bringing it to my attention @spier!

**Review by zkoppert — APPROVED** [review]
🎉 Thanks for the fixes and improvements!! Really appreciate you taking the time to upstream these and help the next user that has to read these docs!

**Pull request merged**